docs/data-sources/instance.md

```diff
@@ -0,0 +1,18 @@
+# <resource name> Resource/Data Source
```

Comment:
Do we need someone from C&I to fill this in?

Reply:
@jtuttle This is still in progress, I'm working on filling this in to the best I can with my limited knowledge of this integration. Maybe once we get this published to the store someone who is more knowledgeable can come back and update these.

Reply:
Ah okay thanks. I'm just going to change the name to include `WIP`, that's our usual method for indicating incomplete PRs.